src/patterns/question-pages/passport.njk

```diff
@@ -20,14 +21,22 @@ layout: layout-example.njk
 
         <form class="form" action="/url/of/next/page" method="post">
 
-          {{ govukInput({
-            label: {
-              "html": '<h3 class="govuk-heading-m govuk-!-mb-r1">Passport number</h3>',
-              "hintText": "For example, 502135326"
-            },
-            id: "passport-number",
-            name: "passport-number"
-          }) }}
+          {% call govukFieldset({
+            "legendHtml": '<h3 class="govuk-heading-m govuk-!-mb-r1">Passport number</h3>',
```

Comment:
Could we instead here set a class for the legend, do we need the nested heading?

Reply:
Adding a heading in a legend helps AT users to navigate using headings. (see [#724](https://github.com/w3c/html/issues/724)), so I see it bringing value and no harm.

Reply:
I _think_ adding a class to the legend would also not work if the question needed a hint (unless the hint styles are defensive enough to be able to be used as a descendant of `govuk-heading-m`.